options/locale/locale_en-US.ini

```
@@ -2331,6 +2331,7 @@ tag.create_success = Tag '%s' has been created.
 
 topic.manage_topics = Manage Topics
 topic.done = Done
+topic.delete = Delete
```

`Delete topic`?
Or is it always clear for users what is supposed to be deleted?
```suggestion
topic.delete = Delete topic
```